tests/app/utils.js

```diff
@@ -8,4 +8,7 @@ const loadMap = async (page) => {
 
 const getNumCities = async (page) => page.locator(CITY_MARKER).count();
 
-export { getNumCities, loadMap };
+const cityDetailsIsVisible = async (page) =>
+  page.$eval(".city-details-popup", (el) => el.style.display === "block");
```

Comment:
Do you expect to use this util with other tests, or only city details? If only city details, it's better to move this function to that file.